src/otx/v2/adapters/torch/mmengine/mmdet/dataset.py

```diff
@@ -15,16 +15,22 @@
 from otx.v2.adapters.torch.mmengine.mmdet.registry import MMDetRegistry
 from otx.v2.api.entities.task_type import TaskType, TrainType
 from otx.v2.api.utils.decorators import add_subset_dataloader
+from otx.v2.api.utils.type_utils import str_to_task_type
 
 SUBSET_LIST = ["train", "val", "test", "unlabeled"]
 
 
-def get_default_pipeline(subset: str = "train", is_semisl: bool = False) -> list:
+def get_default_pipeline(
+    subset: str = "train",
```

Comment:
This should be updated to use `Subset` instead of `str` in the future.

Reply:
I agree, but this should be handled in their parent classes.